lib/js/components/DatePickers/DatePicker/DatePicker.composables.ts

```diff
@@ -130,6 +133,25 @@ export function initFlatpickr({
 			flush: 'post', // Ensure updates happen after DOM changes
 		},
 	);
+	watch(
+		[() => props.date, () => props.startDate, () => props.endDate],
+		() => {
+			if (props.date) {
+				datePicker?.setDate(props.date, false);
+				datePicker?.updateValue(false);
+				datePicker?.jumpToDate(props.date, false);
+			} else if (props.startDate && props.endDate) {
```

Comment:
It seems to work correctly
```suggestion
			} else if (props.startDate || props.endDate) {
```